ui/index.html

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="/media/css/themes.css?v=1">
     <link rel="stylesheet" href="/media/css/auto-save.css?v=1">
     <link rel="stylesheet" href="/media/css/main.css?v=1">
+    <link rel="stylesheet" href="/media/css/auto-save.css?v=1">
```

Comment:
Thanks, but this was already added in a bug fix (two lines above this one) :)

Reply:
Gotcha, I've removed my line!